test-utils/utils/utils.go

```diff
@@ -128,6 +129,35 @@ func (u *Utils) deref(job string, buildNumber int) (directory string, err error)
 	return out, nil
 }
 
+// derefJob reads the file in GCS to figure out where the desired file is.  We
```

Comment:
I'm not sure if I follow this. Which file it reads? Looks like the method accepts a single string (job) and returns directory. Could you explain what it does in terms of 1) what is input and output 2) how input is transformed to output?

Reply:
I realized that this function had no sense. I removed it.